custom_components/midea_ac_lan/light.py

```diff
@@ -35,9 +33,54 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     async_add_entities(devs)
 
 
+def _calc_supported_features(device: Midea13Device) -> LightEntityFeature:
+    supported_features = LightEntityFeature(0)
+    if device.get_attribute(X13Attributes.effect):
+        supported_features |= LightEntityFeature.EFFECT
+    return supported_features
+
+
+def _calc_supported_color_modes(device: Midea13Device) -> set[ColorMode]:
+    # https://github.com/home-assistant/core/blob/c34731185164aaf44419977c4086e9a7dd6c0a7f/homeassistant/components/light/__init__.py#L1278
+    supported = set[ColorMode]()
```

Comment:
I'm not sure with this line， could you help to check and confirm it ?

Reply:
Emm, which line?

Reply:
line 45 set[ColorMode]()

Reply:
This syntax is available in at least Python 3.10, safe to use.

Reply:
After searching, it was probably this:
https://docs.python.org/3.9/library/stdtypes.html#generic-alias-type
Introduced in 3.9

Reply:
> After searching, it was probably this: https://docs.python.org/3.9/library/stdtypes.html#generic-alias-type Introduced in 3.9

thanks, got new syntax.

As I'm not know this syntax, so I just paste it to ChatGPT 3.5 to check and confirm , it said it's error syntax, so i add this comments,  after found your new comments, I just use python3.12 to test and confimed it can works well, so I post it to ChatGPT the answer is error, but ChatGPT still suggest using `supported = set()` or `supported = cast(Set[ColorMode], set())` for compatibility and code clarity .

anyway, homeassiant should not any this issue, so it's safe for us to use.

Reply:
`cast` was too tedious, `supported = set[ColorMode]()` is already clear, or `supported: set[ColorMode] = set()` is also acceptable to me.